tls/s2n_handshake_io.c

```diff
@@ -1052,12 +1053,10 @@ int s2n_negotiate(struct s2n_connection *conn, s2n_blocked_status *blocked)
     notnull_check(conn);
     notnull_check(blocked);
 
-    errno = 0;
```

Comment:
Thy are those moved?

Reply:
The s2n_errno was moved in 85dd8a0e6, I just moved errno reset for consistency and deleted the s2n_errno prior to loop as it is redundant.